### PR TITLE
Product list and Wishlist page

### PR DESCRIPTION
**Pull request by anamkaa**
Added product page

https://user-images.githubusercontent.com/75376715/154311638-b097ae46-6578-422b-b3b7-4ecc92b315bf.mp4

Added wishlist page

https://user-images.githubusercontent.com/75376715/154311645-4e68e33d-5596-4faf-8b07-b730e87ad2d7.mp4

### CODE REVIEW

File: page_layouts/product.html [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+        <div class="card-btn-container gh-btn-container">
+          <button class="card-btn gh-btn gh-btn-primary">Buy Now</button>
+          <button class="card-btn gh-btn gh-btn-secondary">add to cart</button>
+        </div>
+      </div>
+      <!-- card end -->
+
+      <!-- card start -->
+      <div class="gh-card card card-image shadow">
+        <!-- <div class="card-badge gh-green-bg">trending</div> -->
```
Comment (Sameony) [inline page_layouts/product.html]: do remove unnecessary chunks of codes before merging

File: page_layouts/product.html [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+        <div class="h6 gh-filter-clear gh-filter-title">Clear</div>
+      </div>
+      <div class="gh-filter-wrap-col gh-filter-margin">
+        <div class="h5 text-bold gh-filter-title">Price</div>
+        <div class="gh-filter-slider">
+          <span class="gh-slider-text">0</span>
+          <span class="gh-slider-text">500</span>
+          <span class="gh-slider-text">5000</span>
+        </div>
+        <input type="range" name="" id="" class="gh-slider">
```
Comment (coldpigli) [inline page_layouts/product.html]: suggestion not to keep fields blank. Remove it entirely if you don't need it.

File: page_layouts/product.html [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+        <div class="card-btn-container gh-btn-container">
+          <button class="card-btn gh-btn gh-btn-primary">Buy Now</button>
+          <button class="card-btn gh-btn gh-btn-secondary">add to cart</button>
+        </div>
+      </div>
+      <!-- card end -->
+
+      <!-- card start -->
+      <div class="gh-card card card-image shadow">
+        <!-- <div class="card-badge gh-green-bg">trending</div> -->
```
Comment (Sameony) [inline page_layouts/product.html]: same as above,
 and in line 215,242,259 etc

File: stylesheets/page/product.css [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+.gh-filter-text {
+  margin: 0.2rem 0.5rem;
+  font-size: 0.8rem;
+}
+
+.gh-card {
+  border-radius: 0.2rem;
+  box-shadow: rgb(50 50 93 / 25%) 0px 2px 5px -1px,
```
Comment (Sameony) [inline stylesheets/page/product.css]: create variables for colors for better readability

File: stylesheets/page/wishlist.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
+@import url("/stylesheets/style.css");
+
+.gh-wishlist-wrap {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.gh-dismiss {
+  color: #f0f1f3;
```
Comment (Sameony) [inline stylesheets/page/wishlist.css]: same as previous issue, do maintain consistency with the code by using variables

**Review by Sameony — COMMENTED** [review]
suggested a few adjustments

File: stylesheets/page/product.css [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+.gh-btn {
+  border-radius: 0.2rem;
+  font-size: 0.9rem;
+  width: 100%;
+}
+
+.gh-btn-primary {
+  background-color: #afbb8e;
```
Comment (coldpigli) [inline stylesheets/page/product.css]: suggestion to use variables for colors

File: stylesheets/page/wishlist.css (hunk 5 above)
Comment (coldpigli) [inline stylesheets/page/wishlist.css]: Variables instead of hard-coded values

**Review by coldpigli — APPROVED** [review]
Suggested some changes

File: page_layouts/product.html (hunk 1 above)
Comment (anamkaa) [reply page_layouts/product.html]: done , as suggested.

File: page_layouts/product.html (hunk 3 above)
Comment (anamkaa) [reply page_layouts/product.html]: done

File: stylesheets/page/product.css (hunk 6 above)
Comment (anamkaa) [reply stylesheets/page/product.css]: done

File: stylesheets/page/wishlist.css (hunk 5 above)
Comment (anamkaa) [reply stylesheets/page/wishlist.css]: done

File: page_layouts/product.html (hunk 2 above)
Comment (anamkaa) [reply page_layouts/product.html]: done as suggested.

File: stylesheets/page/product.css (hunk 4 above)
Comment (anamkaa) [reply stylesheets/page/product.css]: done

**Pull request merged**